docs/adrs/1040-composite-supports-action-steps.md

```diff
@@ -0,0 +1,44 @@
+**Date**: 2021-04-06
+
+**Status**: Accepted
+
+## Context
+
+We will add support for actions to be referenced, with composite actions.
+
+## Decision
+
+TODO:
+  - Decide recursion limit
+
+## Consequences
+
+TODO: Move these details into a tracking issue instead
+
+- Launch
+  - New feature flag
+  - Move Action policy enforcement from workflow run, to resolve actions endpoint 
+- Runner
+  - New feature flag
+  - Download all nested actions
+    - Decide recursion limit
+    - src/Runner.Worker/ActionManager.cs
+    - Precursor: Remove feature flag DistributedTask.NewActionMetadata
```

Comment:
what does compat look like with GHES... probably fine since we don't need to be that far backward compatible.

what would compat look like for new runner releases with versions of GHES that might not support nested composite... you could unintentionally violate policy if we move the check? that might be okay-ish if we document it?